packages/manager/apps/container/src/container/index.tsx

```diff
@@ -12,28 +12,34 @@ import SSOAuthModal from '@/sso-auth-modal/SSOAuthModal';
 export default function Container(): JSX.Element {
   const { isLoading, betaVersion, useBeta } = useContainer();
   const shell = useShell();
-  const tracking = shell.getPlugin('tracking');
   const isNavReshuffle = betaVersion && useBeta;
+
   if (isLoading) {
     return <></>;
   }
```

Comment:
**nitpick:** Can be moved to return statement. This could fix the reported SonarCloud issue

Reply:
you're right i fixed it, the code is cleaner like that, thanks